src/appleseed/renderer/modeling/surfaceshader/oslsurfaceshader.cpp

```diff
@@ -0,0 +1,246 @@
+
+//
+// This source file is part of appleseed.
+// Visit http://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2015 Esteban Tovagliari, The appleseedhq Organization
```

Comment:
Fix year in copyright notice (it probably should say 2016, or 2015-2016, until I update all notices for the year 2017).